src/Dialogs/HelpDialog.vala

```diff
@@ -0,0 +1,106 @@
+// -*- Mode: vala; indent-tabs-mode: nil; tab-width: 4 -*-
+/*-
+ * Copyright (c) 2018 elementary LLC. (https://elementary.io)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Authored by: Cassidy James Blaede <c@ssidyjam.es>
+ *
+ */
+
+public class HelpDialog : Gtk.Window {
+    private string os = Utils.get_pretty_name ();
+
+    public HelpDialog () {
+        Object (
+            title: _("Help with Dual Booting"),
+            deletable: true,
+            resizable: false
+        );
+    }
+
+    construct {
+        var header = new Gtk.HeaderBar ();
+        header.set_show_close_button (true);
+        var header_context = header.get_style_context ();
+
+        var dual_booting = new Gtk.Label ("""<b>Create Partitions</b>
```

Comment:
I know I wrote this whole dialog, but I'm not super happy with it. It's big and untranslatable. There's probably a better way to solve this, and this is a prime candidate to be broken off into its own PR.